lib/services/upload-service.ts

```diff
@@ -0,0 +1,49 @@
+import * as uuid from "uuid";
+
+import { BUILD_SERVICE_NAME, HTTP_METHODS } from "../constants";
+
+export class UploadService implements IUploadService {
+
+	protected serviceName = BUILD_SERVICE_NAME;
+
+	constructor(private $buildCloudService: IBuildCloudService,
+		protected $cloudRequestService: ICloudRequestService,
```

Comment:
We don't need the `$cloudRequestService`.

Reply:
🎣